apps/browser/qml/pages/components/HistoryList.qml

```diff
@@ -20,42 +20,20 @@ SilicaListView {
     property bool showDeleteButton
     property bool menuClosed
 
-    signal load(string url, string title)
+    signal load(string url, string title, bool newTab)
+    signal saveBookmark(string url, string title)
+
+    onLoad: webView.privateMode = !webView.privateMode
```

Comment:
You'll need a bit of logic here to avoid switching modes when the user presses on a history item (rather than selecting "new tab" from the menu).

Reply:
Thanks, did not notice), added a condition.